clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -449,18 +449,19 @@ import           Clash.Annotations.Primitive
   (Primitive(InlineYamlPrimitive), HDL(..), hasBlackBox)
 import           Clash.Class.Num        (SaturationMode(SatBound), satSucc)
 import           Clash.Explicit.Signal  (KnownDomain, Enable, register, fromEnable)
+import           Clash.Explicit.BlockRam.Model (tdpbramModel)
```

Comment:
Sure, I'll sort them! I also think we should adopt Bittide's way of organizing inputs. Basically the same as here, but:

* Yeet qualified imports at the end in their own group
* Don't align

This makes imports sortable with tools 90% of the time (instead of the 10% we have now).

Reply:
I'd be in favour of changing our style to the Bittide idea. But please with the same groupings inside the `qualified` supergroup as are currently in our style guide, I think that does serve a good purpose.